docs/tutorials/replicated-stateful-application/mysql-statefulset.yaml

```diff
@@ -0,0 +1,144 @@
+apiVersion: apps/v1beta1
```

Comment:
Since this initial comment thread, I've also added a client service in addition to the headless one. So now I have two services in `mysql-services.yaml`. I also have `mysql-configmap.yaml`, so 3 files total. I think if it was just a StatefulSet and one headless service, I would agree with putting them together to get to the one-file ideal. However, with 2 services and a ConfigMap, I feel like the separation adds enough clarity to be worth the extra files.